### ISSUE TITLE
Inference output does not cover complete layout

### ISSUE THREAD

**Issue by poojitha0892** [ultralytics/ultralytics#17373]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Ultralytics YOLO Component

Other

### Bug

I trained yolo11x model on manually annotated document layout dataset for 300 epochs. When I'm inferencing on test dataset, I see a recurring issue - in any one of the layout (for suppose if its a sentence) its not detecting complete sentence, instead it's just detecting center part of the sentence. Here's a sample image - 
![PMC3440578_00007_issue](https://github.com/user-attachments/assets/03cf8c65-0f3f-4bca-aedd-241fe8484759)
I'm struck with this issue, can someone suggest a way how to resolve this. Thanks in advance!

### Environment

python 3.10.0 ,CUDA Version: 12.2 , training batch size 80, training dataset contains ~30k images

### Minimal Reproducible Example

-

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @poojitha0892, thank you for reaching out and for your interest in Ultralytics 🚀! This is an automated response, but don't worry, an Ultralytics engineer will also assist you soon.

If this is a 🐛 Bug Report, it seems like you've encountered an issue with inference performance. To assist us in helping you, could you please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) showcasing the problem? This will help us debug it more effectively.

We encourage you to visit our [Docs](https://docs.ultralytics.com) which contain many helpful [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples. If you are training custom models, verify that you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Meanwhile, you might want to ensure you're using the latest version of our package. You can upgrade using the following command:

```bash
pip install -U ultralytics
```

Furthermore, joining our community could provide additional insights and solutions. Consider engaging with us on [Discord](https://ultralytics.com/discord) 🎧 for real-time chat, [Discourse](https://community.ultralytics.com) for in-depth discussions, or our [Subreddit](https://reddit.com/r/ultralytics) for sharing and discovering knowledge within the community.

## Environments

Here’s a reminder about the environments where YOLO can run smoothly:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

If the badge is green, the current [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are passing, ensuring proper operation across different platforms.

Stay tuned, and thank you for your patience as we work together to resolve your issue! 😊

**Comment by Y-T-G** [comment]
You can check [this](https://github.com/ultralytics/ultralytics/issues/10229#issuecomment-2437045639). Use the same yaml.

**Comment by poojitha0892** [comment]
> You can check [this](https://github.com/ultralytics/ultralytics/issues/10229#issuecomment-2437045639). Use the same yaml.
My task is object detection and I want yolov11 yaml

**Comment by Y-T-G** [comment]
You can use the P6 yaml from here

https://github.com/ultralytics/ultralytics/blob/0f640086dd5a075dbf63722d99a73f4720dde468/ultralytics/cfg/models/11/yolo11-p6.yaml

**Comment by Y-T-G** [comment]
You can try the YOLOv8 yaml I linked then. The one in my link is for detection.

**Comment by poojitha0892** [comment]
You want me to try training yolov8 model??

**Comment by Y-T-G** [comment]
This one

https://github.com/ultralytics/ultralytics/issues/16895#issuecomment-2413016050

**Comment by Y-T-G** [comment]
You can also try installing this branch and then train.

https://github.com/ultralytics/ultralytics/issues/11634#issuecomment-2367471426
